app/controllers/api/v1/employees_controller.rb

```diff
@@ -0,0 +1,49 @@
+class Api::V1::EmployeesController < ApplicationController
+  before_action :find_employee, only: [:edit, :update, :destroy]
+
+  def index
+    @employees = Employee.all
+  end
+
+  def show
+    @employee = Employee.find(params[:id]).decorate
```

Comment:
You have `find_employee`
mention it in the `before_action`
then just `@employee.decorate` in `show`